site/components/map/Map.tsx

```diff
@@ -44,6 +44,50 @@ const Map = ({
     const [features, setFeatures] = useState<{ [key: string]: PolygonFeature }>({});
     const [markerData, setMarkerData] = useState<Stop[]>([]);
     const [selectAll, setSelectAll] = useState<boolean>(true);
+    const [popupInfo, setPopupInfo] = useState<Partial<Stop>>({});
+
+    const handleMouseEnter = useCallback(
+        (id: string) => {
+            const stopsOnMap = searched && selected ? [...selected, ...searched] : [];
```

Comment:
are searched and selected ever empty arrays?

Reply:
they won't be empty when i do services